### PR TITLE
Id 80 frontend jwt authentication

### PR DESCRIPTION
**Pull request by S-zeyu**
Frontend login authentication using backend api done - as well as sending over jwt token for each request on loading subsequent webpages

### CODE REVIEW

**Review by leeswshawn — APPROVED** [review]

File: account-service/src/main/java/com/service/accountservice/config/JwtAuthenticationFilter.java [hunk 1]
```diff
@@ -62,7 +62,5 @@ protected void doFilterInternal(
                 SecurityContextHolder.getContext().setAuthentication(usernamePasswordAuthenticationToken);
             }
         }
-        filterChain.doFilter(request,response);
```
Comment (leeswshawn) [inline account-service/src/main/java/com/service/accountservice/config/JwtAuthenticationFilter.java:65]: Unintentional removal, please revert this change

**Review by leeswshawn — CHANGES_REQUESTED** [review]
Unintentional removal of code

File: account-service/src/main/java/com/service/accountservice/config/JwtAuthenticationFilter.java (hunk 1 above)
Comment (S-zeyu) [reply account-service/src/main/java/com/service/accountservice/config/JwtAuthenticationFilter.java:65]: will add it on the next commit

File: account-service/src/main/java/com/service/accountservice/Controller/demo.java [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+
 @RestController
 @RequestMapping("/api/v1/demo-controller")
 public class demo {
-
+    @CrossOrigin(origins = "http://localhost:3000")
     @GetMapping
     public ResponseEntity<String> sayHello() {
-        return ResponseEntity.ok("Hello from secured endpoint");
+        return ResponseEntity.ok("access granted");
```
Comment (shaysnutss) [inline account-service/src/main/java/com/service/accountservice/Controller/demo.java:17]: Just for future reference , this won’t be the actual api that will first be called when logging in, and entering the application. It probably would be the api that returns data from visualisation. But works for now

**Review by shaysnutss — APPROVED** [review]

**Review by leeswshawn — APPROVED** [review]
